src/app/DIAmeterApplication.cpp

```diff
@@ -73,241 +80,871 @@ int DIAmeterApplication::main(const vector<string>& input_files, const string in
-
-		  // The denominator used in the Tailor score calibration method
-		  double quantile_score = getTailorQuantile(&match_arr2);
-		  // carp(CARP_DETAILED_DEBUG, "Tailor quantile_score:%f", quantile_score);
-
-		  TideMatchSet::Arr match_arr(nCandPeptide);
-		  for (TideMatchSet::Arr2::iterator it = match_arr2.begin(); it != match_arr2.end(); ++it) {
-			  /// Yang: I cannot understand here that the peptide index and the rank is complement.
-			  /// More attention is needed.
-			  int peptide_idx = candidatePeptideStatusSize - (it->second);
-			  if ((*candidatePeptideStatus)[peptide_idx]) {
-				  TideMatchSet::Scores curScore;
-			      curScore.xcorr_score = (double)(it->first / XCORR_SCALING);
-			      curScore.rank = it->second;
-			      curScore.tailor = ((double)(it->first / XCORR_SCALING) + 5.0) / quantile_score;
-			      match_arr.push_back(curScore);
-			      // carp(CARP_DETAILED_DEBUG, "peptide_idx:%d \t xcorr_score:%f \t rank:%d", peptide_idx, curScore.xcorr_score, curScore.rank);
-			  }
-		  }
-		  TideMatchSet matches(&match_arr, highest_ms2_mz);
-		  if (!match_arr.empty()) { reportDIA(output_file, origin_file, *sc, active_peptide_queue, proteins, locations, &matches, &ms1scan_intensity_rank_map, &peptide_predrt_map); }
+  if (Params::GetBool("psm-filter")) {
+	  stringstream param_ss;
+	  param_ss << "diameter-search.filtered." << getCoeffTag() << ".txt";
+	  output_file_name_filtered_ = make_file_path(param_ss.str().c_str());
+  }
 
-	  }
 
-	  delete spectra;
-	  delete sc_index;
-	  for (map<int, pair<double*, double*>>::const_iterator i = ms1scan_intensity_rank_map.begin(); i != ms1scan_intensity_rank_map.end(); i++) {
-		  delete[] (i->second).first;
-		  delete[] (i->second).second;
+  // Extract all edge features
+  if (!FileUtils::Exists(output_file_name_unsorted_) || Params::GetBool("overwrite") /* */) {
+	  carp(CARP_DEBUG, "Either file exists or it needs to be overwrite:%s", output_file_name_unsorted_.c_str());
+
+	  ofstream* output_file = create_stream_in_path(output_file_name_unsorted_.c_str(), NULL, Params::GetBool("overwrite"));
+	  TideMatchSet::writeHeadersDIA(output_file, Params::GetBool("compute-sp"));
+
+	  map<string, double> peptide_predrt_map;
+	  getPeptidePredRTMapping(&peptide_predrt_map);
+
+	  vector<InputFile> ms1_spectra_files = getInputFiles(input_files, 1);
+	  vector<InputFile> ms2_spectra_files = getInputFiles(input_files, 2);
+
+	  // Loop through spectrum files
+	  for (pair<vector<InputFile>::const_iterator, vector<InputFile>::const_iterator> f(ms1_spectra_files.begin(), ms2_spectra_files.begin());
+		 f.first != ms1_spectra_files.end() && f.second != ms2_spectra_files.end(); ++f.first, ++f.second) {
+
+		 string ms1_spectra_file = (f.first)->SpectrumRecords;
+		 string ms2_spectra_file = (f.second)->SpectrumRecords;
+		 string origin_file = (f.second)->OriginalName;
+
+		 // load MS1 and MS2 spectra
+
+		 // changing from loadMS1SpectraOld to loadMS1SpectraNew step-by-step
```

Comment:
based off these comments it sounds like you should remove any fxn that has "old" in it and remove the word "new" in the fxn that remains.